docs/plots/backends.rst

```diff
@@ -0,0 +1,362 @@
+
+Most plotting backends accept more than just the backend-independent options
+listed above. For example, :term:`matplotlib` allows `many different ways to
+specify colors to cover the entire RGB range
+<https://matplotlib.org/stable/tutorials/colors/colors.html>`_, such as
+``color=(.3, .4, .5, .2)`` or ``color='xkcd:eggshell'``. The Sherpa plotting
+methods will pass any value to the underlying backend, but only the options in
+the backend independent list are guaranteed to work in every case. For example, 
+
+  >>> from sherpa.data import Data1D
+  >>> from sherpa.plot import DataPlot
+  >>> d = Data1D('example data', [1, 2, 3], [3, 2, 5])
+  >>> dplot = DataPlot()
+  >>> dplot.prepare(d)
+  >>> dplot.plot(markerfacecolor='xkcd:eggshell')
+
+will succeed with the `~sherpa.plot.pylab_backend.PylabBackend`, but raise an
+error if the active backend is `~sherpa.plot.bokeh_backend.BokehBackend`. In
+contrast, ``color='k'`` is also not understood by bokeh natively, but because it is on
+the backend-independent list, Sherpa will translate ``'k'`` to a form that bokeh
+does understand (``'black'`` in this case).
+
+Backends may also accept additional keywords to specify more plotting properties
+such as the transparency of an element or an URL that is opened when clicking on
+an element. Those can simply be passed to the Sherpa plotting command, which
+will pass them through to the plotting backend:
+
+  >>> from sherpa.data import Data1D
+  >>> from sherpa.plot import DataPlot
+  >>> d = Data1D('example data', [1, 2, 3], [3, 2, 5])
+  >>> dplot = DataPlot()
+  >>> dplot.prepare(d)
+  >>> dplot.plot(url='https://www.example.com')
+
+Since Sherpa does not process those options itself, but just passes them on to
+the underlying backend module, they are not documented here - see the
+documentation of the specific plotting module for details. Also, they will fail and
+raise an error if the plotting backend in use does not understand the ``url`` keyword.
+
+In some cases, the Sherpa plotting commands create several visualization
+elements at the same time (lines, symbols, error bars, axes, labels). This makes
+using Sherpa classes convenient, but it also means that the plotting functions
+do not offer options to customize each and every part. In general, the plotting
+functions pass color, linestyle etc. to the elements that describes the data
+(line, marker) and generate labels or axes grids using default settings. Backend
+specific code can be used to change the properties of the current figure after
+the Sherpa plotting.
+
+Backend interface
+=================
+
+.. note::
+
+   This section is mostly relevant for developers or advanced users who write new
+   Sherpa plot classes or new backends.
+
+This section describes the API that all Sherpa backends offer to explain how to
+use it and why it was designed this way. See `sherpa.plot.backend.BaseBackend`
+for a complete listing of the calling signature for each function. 
```

Comment:
The `*Backend` are drawn as code, but not as links here - presumably for the same reason as the `*Backend` cases above https://github.com/sherpa/sherpa/pull/1382/files#r877545827